api/tests/integration/infrastructure/utils/pdf/certification-attestation-pdf_test.js

```diff
@@ -105,42 +105,58 @@ describe('Integration | Infrastructure | Utils | Pdf | Certification Attestation
   it('should generate a page per certificate', async function () {
     // given
     const resultCompetenceTree = domainBuilder.buildResultCompetenceTree();
-    const certificateWithCleaAndPixPlusDroitMaitre = domainBuilder.buildCertificationAttestation({
-      id: 1,
-      firstName: 'Jean',
-      lastName: 'Bon',
-      resultCompetenceTree,
-      certifiedBadges: [{ partnerKey: PIX_EMPLOI_CLEA_V3 }, { partnerKey: PIX_DROIT_MAITRE_CERTIF }],
-    });
-    const certificateWithCleaAndPixPlusDroitExpert = domainBuilder.buildCertificationAttestation({
-      id: 2,
-      firstName: 'Harry',
-      lastName: 'Covert',
-      resultCompetenceTree,
-      certifiedBadges: [{ partnerKey: PIX_EMPLOI_CLEA_V3 }, { partnerKey: PIX_DROIT_EXPERT_CERTIF }],
-    });
-    const certificateWithoutCleaNorPixPlusDroit = domainBuilder.buildCertificationAttestation({
-      ...certificateWithCleaAndPixPlusDroitMaitre,
-      id: 2,
-      firstName: 'Marc',
-      lastName: 'Decaffé',
-      cleaCertificationImagePath: null,
-      pixPlusDroitCertificationImagePath: null,
-      certifiedBadges: [],
-    });
+    const certificateWithCleaAndPixPlusDroitMaitreAndWithoutProfessionalizingMessage =
+      domainBuilder.buildCertificationAttestation({
+        id: 1,
+        firstName: 'Jean',
+        lastName: 'Bon',
+        resultCompetenceTree,
+        certifiedBadges: [{ partnerKey: PIX_EMPLOI_CLEA_V3 }, { partnerKey: PIX_DROIT_MAITRE_CERTIF }],
+      });
```

Comment:
Expliciter le critère de date caché à 2018 < 2022 dans `buildCertificationAttestation`
exemple avec dayjs https://day.js.org/docs/en/manipulate/subtract
```suggestion
      const PROFESSIONALIZING_VALIDITY_START_DATE = new Date('2022-01-01').getTime();
      const deliveredBeforeStartDate = PROFESSIONALIZING_VALIDITY_START_DATE.substract(1,'day');
    const certificateWithCleaAndPixPlusDroitMaitreAndWithoutProfessionalizingMessage =
      domainBuilder.buildCertificationAttestation({
        id: 1,
        firstName: 'Jean',
        lastName: 'Bon',
        resultCompetenceTree,
        deliveredAt: deliveredBeforeStartDate,
        certifiedBadges: [{ partnerKey: PIX_EMPLOI_CLEA_V3 }, { partnerKey: PIX_DROIT_MAITRE_CERTIF }],
      });
```

Reply:
Done.